inspirehep/dojson/conferences/fields/bd1xx.py

```diff
@@ -33,40 +35,70 @@
 @utils.for_each_value
 def acronym(self, key, value):
     """Conference acronym."""
+    def append_to_self(title):
+        """Make a dictionary and append to self."""
+        titles = {}
+        titles['title'] = title
+        titles['subtitle'] = value.get('b')
```

Comment:
Just to be sure, a title list or tuple will always have a single subtitle? Are you sure you don't need to zip `value.get('b')` together with `value.get('a')` in the case they are both lists?